test.js

```diff
@@ -223,6 +223,17 @@ test('should ignore digits', (t) => {
     }, t.ifErr)
 })
 
+test('should ignore times', (t) => {
+  t.plan(1)
+
+  retext()
+    .use(retextSpell, enGb)
+    .process('2:41pm')
```

Comment:
```suggestion
    .process('Let’s meet at 2:41pm.')
```

A more real example.